maintenance/enableDiscussionsWithNoForums.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Enables Discussions on the given wikis, after verifying there are no threaded forum posts on each wiki
+ * Usage: php enableDiscussionsWithNoForums.php <listfile>
+ * Where:
+ * 	<listfile> is a file where each line is a site ID where we want to enable discussions
+ *
+ * @ingroup Maintenance
+ */
+use Swagger\Client\ApiException;
+use Swagger\Client\Discussion\Api\SitesApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
+require_once( __DIR__ . '/Maintenance.php' );
+
+class EnableDiscussionsWithNoForums extends Maintenance {
+
+	const CURL_TIMEOUT = 5;
+	const SERVICE_NAME = 'discussion';
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Enables Discussions on given wikis with no threaded forum posts";
+		$this->addArg( 'listfile', 'File with site IDs to enable discussions on, separated by newlines.' );
+	}
+
+	public function execute() {
+		$fh = fopen( $this->getArg() , 'r' );
+
+		if ( !$fh ) {
+			$this->error( 'Unable to read from input file, exiting', true );
+		}
+
+		$schwartzToken = F::app()->wg->TheSchwartzSecretToken;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$schwartzToken" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)